src/Azure.Functions.Cli/Kubernetes/KubernetesHelpers.cs

```diff
@@ -680,19 +677,6 @@ private static string GetKedaTrigger(string triggerType)
             }
         }
 
-        private static async Task<bool> HasKeda()
```

Comment:
These methods were never used to figured it was good to remove them.

Happy to re-add if you want to.